web/vital_records/views/common.py

```diff
@@ -107,9 +107,24 @@ class StatementView(EligibilityMixin, ValidateRequestIdMixin, UpdateView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context["page_title"] = f"Replacement {self.object.type} record"
+
+        record_type = self.object.type
+
+        context["page_title"] = f"Replacement {record_type} record"
         previous_route = Routes.app_route(Routes.request_type)
         context["previous_url"] = reverse(previous_route, kwargs={"pk": self.object.pk})
+
+        if record_type == "death":
+            context["authorized_copy_explanation"] = (
+                "To get an authorized copy, you must be the individual legally authorized to make this request — like a "
```

Comment:
Should this be:

> To get an authorized copy, you must be ~the~ **an** individual legally authorized...

cc @mrose914

Reply:
Updated in https://github.com/Office-of-Digital-Services/cdt-ods-disaster-recovery/pull/424/commits/a5d8e733ace68a773a377679c6fcd915198c9097